lib/main_common.pm

```diff
@@ -2273,6 +2273,13 @@ sub load_security_tests_crypt_tool {
     loadtest "console/openvswitch_ssl";
 }
 
+sub load_security_tests_crypt_libtool {
+    loadtest "console/consoletest_setup";
```

Comment:
I think the ' loadtest "console/consoletest_setup";' is needed for each part of security test, maybe you can think to load it in load_security_tests while not loaded each part.

Reply:
@lemon-suse Thanks, it is good suggestion, I will directly load **load_security_console_prepare** here and retest again.

Reply:
DONE